jwst/pipeline/calwebb_coron3.py

```diff
@@ -76,44 +74,37 @@ def process(self, input):
             psf_models.append(psf_input)
             psf_input.close()
 
-        # Call the stack_refs step to stack all the PSF images into
-        # a single CubeModel
+        # Stack all the PSF images into a single CubeModel
         psf_stack = self.stack_refs(psf_models)
         psf_models.close()
 
         # Save the resulting PSF stack
-        output_file = mk_prodname(self.output_dir, prod['name'], 'psfstack')
-        log.info('Saving psfstack file %s', output_file)
-        psf_stack.save(output_file)
+        psf_stack.meta.filename = prod['name']
+        self.save_model(psf_stack, suffix='psfstack')
 
         # Call the sequence of steps align_refs, klip, and outlier_detection
         # once for each input target exposure
         resample_input = datamodels.ModelContainer()
         for target_file in targ_files:
 
             # Call align_refs
-            log.debug('Calling align_refs for member %s', target_file)
+            self.log.debug('Calling align_refs for member %s', target_file)
             psf_aligned = self.align_refs(target_file, psf_stack)
 
             # Save the alignment results
-            filename = mk_filename(self.output_dir, target_file, 'psfalign')
-            log.info('Saving psfalign file %s', filename)
-            psf_aligned.save(filename)
+            self.save_model(psf_aligned, suffix='psfalign')
 
             # Call KLIP
-            log.debug('Calling klip for member %s', target_file)
-            #psf_sub, psf_fit = self.klip(target_file, psf_aligned)
+            self.log.debug('Calling klip for member %s', target_file)
             psf_sub = self.klip(target_file, psf_aligned)
```

Comment:
Is klip returning 2 data models (psf_sub and psf_fit)? If so, what happens on this line when you're capturing only 1 returned object? Does the 2nd one just vanish into the ether?

Reply:
Never mind. I see that I in fact was the one to update the klip step some time ago so that it only returns a single data model and old line 105 was also commented out at that time. So new line 99 is correct.